src/01-functions.js

```diff
@@ -168,7 +190,9 @@ function joinStringsWithSpaces() {}
  *  getFirstAndLastCharacter("upwards");
  *  //> "us"
  */
-function getFirstAndLastCharacter() {}
+function getFirstAndLastCharacter(a) {
+  return a.charAt(0) + a.charAt(a.length-1);
```

Comment:
Nice work here - but remember we can index strings like we index arrays.  What would ` console.log(a[0]) ` log?